exonum/src/node/mod.rs

```diff
@@ -23,7 +23,15 @@ pub use self::{
 };
 
 // TODO: Temporary solution to get access to WAIT constants. (ECR-167)
```

Comment:
I suppose this TODO is outdated as well and can be removed.

Reply:
Don't think so, we still have to think whether it's applicable to have those constants in public.